### ISSUE TITLE
Create a Simple Basic Webpage using html

### ISSUE THREAD

**Issue by Aniruddha775** [Aniruddha775/Hacktoberfest2021-Contribute#15]

**Comment by ritesh423** [comment]
I would like to work on it. Can you assign it to me.

**Comment by ayushyadav9** [comment]
I want to work on it, will you please assign it to me?

**Comment by shubhnanda** [comment]
hello 
can I please work on the issue.
I want to contribute and will make a interesting web page

**Comment by manasvip01** [comment]
Solved it #30

**Comment by Gaurav-Verma07** [comment]
Hey, I also would like to do this.
Please assign this task to me

**Comment by PreetiBaskey** [comment]
Hey... !!! , here is some of my work, if you like it, kindly accept my PR and assign me the issue.
Thank You.

Login Page : https://user-images.githubusercontent.com/38905783/136644821-4a389b66-58c1-41d3-82aa-633a3f0c9d9e.jpg

Landing Page : https://user-images.githubusercontent.com/38905783/136644828-df4cd4d6-931b-4c36-b083-81ecf271b0ed.jpg

Location Access Page : https://user-images.githubusercontent.com/38905783/136644845-87045a1b-e6e3-4858-b079-0bbe1b7d6b9b.jpg

Check In Page : https://user-images.githubusercontent.com/38905783/136644856-64aa03ce-5aee-4274-8080-b7ef01d0728d.jpg

Success Page : https://user-images.githubusercontent.com/38905783/136644863-ae400c9f-9fb8-4d1c-b47a-c5c3005dfa14.jpg

**Comment by abdul-sohail24** [comment]
Hey there, I would like  to work on it, could you please assign it to me

**Comment by its-hrs** [comment]
`<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Harsh_JQuery8</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
![127 0 0 1_5500_JQuery_JQuery8Demo html](https://user-images.githubusercontent.com/82232032/137789807-b3e660b8-eb11-4f86-aff5-a45111ac6710.png)
![Screenshot (338)](https://user-images.githubusercontent.com/82232032/137789867-02e43d8a-3432-4d7a-a72f-e2c93e66de0d.png)
![127 0 0 1_5500_JQuery_JQuery8Demo html](https://user-images.githubusercontent.com/82232032/137789932-02bca5ca-f72e-4b96-918e-5c0d9abae517.png)

  </head>

  <body>
    <div class="container">
      <div class="row">
      <div class="col-lg-8 col-md-6  col-6">
    <a href="http://dsc.du.ac.in/" target="_blank"> <img class="img-fluid" id="top" src="/JQuery/img/DSC.png" alt="DSC College"> </a>
    </div>
    <div class="col-lg-4 col-md-6  col-6">
    <a href="https://www.facebook.com/Dyal-Singh-College-University-of-Delhi-111650217206995" target="_blank"><i style="float: right; margin-top: 0.6%; padding: 10px;" class="fab fa-facebook fa-2x" id="fb" ></i></a>
    <a href="https://twitter.com/DSC_DU" target="_blank"><i style="float:right; margin-top: 0.6%; padding:10px;" class="fab fa-twitter fa-2x" id="tweet" ></i></a>
    <a id="me" href="https://github.com/its-hrs" target="_blank"><i style="padding: 12px; float:right " class="fab fa-github fa-2x" id="github"></i></a>
    <a href="http://dsc.du.ac.in/" target="_blank" style="color: white; font-size: 0.9rem; text-decoration: none;"><b>Contact Us</b></a>
    </div>
    </div>
    </div>

   <nav class="navbar sticky-top navbar-expand-lg p-3 navbar-dark bg-primary nav_style">
    <div class="container-fluid">
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <ul class="navbar-nav ml-auto pr-5 text-capitalize ">
          <li class="nav-item">
            <a class="nav-link" href="#" id="homebtn">Home</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="#" id="feebtn">Fee</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="#" id="elgbtn">Eligibility</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="#" id="paprbtn">Papers</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="#" id="scopbtn">Scopes</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="#" id="comestbtn">COMEST</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>

   <div class="container-fluid pt-5 pb-5" id="change"> 

    <div class="container" id="homepage">
      <div class="row">
        <div class="col-lg-12  col-md-12  col-12 mt-5">
          <h1>B.Sc.(Hons.) Computer Science</h1>
                <p>
                  The Department of Computer Science offers 3-year B.Sc. (Hons.) Computer
                  Science programme through constituent colleges of the University of Delhi.
                  With the new advancements in the field of computers and in a time when
                  there is a boom in the IT industry, the University of Delhi has introduced
                  B.Sc (Hons.) Computer Science, a 3-year undergraduate programme, for the
                  tech-savvy youth. The colleges are contributing to the development of
                  Information Technology by offering this programme with the help of
                  efficient and highly qualified teachers and through well-equipped computer
                  labs. The programme provides rigorous foundations of the concepts of
                  Computer Science and Information Technology. In the final year, students
                  also get an opportunity to do project work. Hence the combination of the
                  concepts and training of software tools equip the students to adapt to
                  ever-changing technology
                </p>
             </div>   
      </div>
    </div>
  </div>

    
  <p>For more details Visit: <a href="http://cs.du.ac.in/programmes/bsc/" target="_blank">Click here</a></p>

    <script src="https://kit.fontawesome.com/aba415448e.js" crossorigin="anonymous"></script>
    <script>
      function styling() {
        $("p").addClass("cont1");

        
        $("#homebtn").on( "click", function(e){
          e.preventDefault();
          var xhr = new XMLHttpRequest(); 
            xhr.onload = function() { 
            if(xhr.status === 200) { 
            document.getElementById("change").innerHTML = xhr.responseText ; 
            }
            };
            xhr.open('GET', 'home.html' , true);
            xhr.send(null);
          });   

        $("#feebtn").on( "click", function(e){
        e.preventDefault();
        var xhr = new XMLHttpRequest(); 
            xhr.onload = function() { 
            if(xhr.status === 200) { 
            document.getElementById("change").innerHTML = xhr.responseText ; 
            }
            };
            xhr.open('GET', 'fee.html' , true);
            xhr.send(null);
          });

        $("#elgbtn").on( "click", function(e){
        e.preventDefault();
        var xhr = new XMLHttpRequest(); 
            xhr.onload = function() { 
            if(xhr.status === 200) { 
            document.getElementById("change").innerHTML = xhr.responseText ; 
            }
            };
            xhr.open('GET', 'eligibility.html' , true);
            xhr.send(null);
          });

        $("#scopbtn").on( "click", function(e){
        e.preventDefault();
        var xhr = new XMLHttpRequest(); 
            xhr.onload = function() { 
            if(xhr.status === 200) { 
            document.getElementById("change").innerHTML = xhr.responseText ; 
            }
            };
            xhr.open('GET', 'scope.html' , true);
            xhr.send(null);
          }); 


          $("#paprbtn").click(function(){
          $("#change").load("paper.html")
          });


          $("#comestbtn").on( "click", function(e){
          e.preventDefault();
          var xhr = new XMLHttpRequest(); 
            xhr.onload = function() { 
            if(xhr.status === 200) { 
            document.getElementById("change").innerHTML = xhr.responseText ; 
            }
            };
            xhr.open('GET', 'comest.html' , true);
            xhr.send(null);
          });

          $(document).ready(function(){
          $(".nav").on("click","button",function(){
          $(".active").removeClass("active");
          $(this).addClass("active");
    })
});

        
        $("h1").addClass("heading");
        $("h2").addClass("heading2");
        $('#homepage').addClass("newdiv");
        $('p').find('a').css("color","black");
        $("#mybtn").css("display", "none");
        
      }
    </script>

    <div class="container text-center">
    <a href="#top"><button id="mybtn" onclick="styling()" type="button" class="btn btn-primary btn-lg">Change Style</button></a></div>
    <hr />
    <div style="text-align: center; height: 0vh; width: 100%;">
      <div class=" footer_style bg-primary text-white container-fluid text-center">
       &copy; Harsh Saxena | <a id="me" href="https://github.com/its-hrs" target="_blank"><i style="padding: 5px;" class="fab fa-github" id="github"></i></a>
        <a href="https://www.facebook.com/PotterHead9347" target="_blank"><i style="padding: 5px" class="fab fa-facebook" id="fb" ></i></a>
        <a href="https://twitter.com/Saxenaboy_123" target="_blank"><i style="padding: 5px;" class="fab fa-twitter" id="tweet"></i></a><br />All Rights Reserved | 2021-22
      </div>
 
      
    </div>

        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
  </body>
</html>
`
